### PR TITLE
Fix: Typo and feedback on support panel

### PR DESCRIPTION
**Pull request by joethreepwood**
## Problem

There's a typo in the support panel. This PR fixes that.

For clarity, I'll also summarize my feedback on this panel which is in this thread: https://posthog.slack.com/archives/C02E3BKC78F/p1712671176930389

> I’m catching up on things here, but I also think that we shouldn’t direct to GitHub from here, and also that there’s overlap between some items.
> 
> For bugs, I think we should handle these in Zendesk. This means we get more information on the ticket, but more importantly means we’ll also make things easier for users. Currently, we’re basically telling users that they have to sign-up and login to GitHub to raise a bug report. That may be fine for developers (even if it’s more work for them) but not for the significant %age of non-developers.
> 
> I agree that paying users should get directed to us more quickly. They are, afterall, paying for the privilege.
> I also think there’s some overlap between ‘What we’re building’, ‘Vote on our roadmap’, and ‘Request a feature’. This is probably on W&D timeline already, but I’d assume it’s easiest for have all of this in just one place where users can see all the above?

## Changes

👉 _Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review._

### CODE REVIEW

File: frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx [hunk 1]
```diff
@@ -184,7 +184,7 @@ export const SidePanelSupport = (): JSX.Element => {
 
                     <Section title="Ask the community">
                         <p>
-                            Questions about features, how to's, or use cases? There are thousands of discussions in our
+                            Questions about features, how tos, or use cases? There are thousands of discussions in our
```
Comment (corywatilo) [inline frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx:187]: Actually I think the _correct_ way is to hyphenate it ;)

Comment (joethreepwood) [reply frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx:187]: You're correct, it's a compound noun without a possessive

```suggestion
                            Questions about features, how-tos, or use cases? There are thousands of discussions in our
```

**Review by MarconLP — APPROVED** [review]

File: frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx [hunk 2]
```diff
@@ -211,9 +211,6 @@ export const SidePanelSupport = (): JSX.Element => {
                                     </LemonBanner>
                                 </Section>
                             ) : null}
-
-                            <Section title="Contact us">
-                                <p>Can't find what you need in the docs?</p>
```
Comment (MarconLP) [inline frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx:216]: Why are we removing the section title here? We would need to remove the closing `</Section>` as well, but I guess this is a merge error?

**Review by MarconLP — CHANGES_REQUESTED** [review]

File: frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx (hunk 2 above)
Comment (joethreepwood) [reply frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx:216]: Yeah, this is a merge error. Let me just close this and open a new one.

**Pull request closed without merging**